docs/api/command-line-switches.md

```diff
@@ -241,6 +241,12 @@ Force using discrete GPU when there are multiple GPUs available.
 
 Force using integrated GPU when there are multiple GPUs available.
 
+### --xdg-portal-required-version=`version`
+
+Sets the minimum required version of XDG portal implementation to `version`
+inorder to use the portal backend for file dialogs on linux. Current
+default is set to `4`.
```

Comment:
Who is the intended consumer for this switch? Application developers, users, or both? For the recent bump to v4, is this so application users can switch back the behavior if they require it?

I also wonder if we should consider reverting the default to v3. #43570 contained an undocument breaking change in behavior that applications may not be expecting.

Reply:
I don't think we should revert the default, since it looks like the change was accepted upstream (see https://chromium-review.googlesource.com/c/chromium/src/+/5786982), but we probably should document this as a breaking change, at least retroactively, so people know

Reply:
@VerteDinde I do think (in agreement w/ deepak below)  that we should change back the default, since as of now v4 is so rarely available that every app might be required to float this flag for potentially years.